cmd/gitops/root/cmd.go

```diff
@@ -85,6 +88,31 @@ func RootCmd() *cobra.Command {
 				fmt.Fprintf(os.Stderr, "Error: %v\n", err)
 				os.Exit(1)
 			}
+
+			_, err = config.GetConfig(false)
+			if err != nil {
+
```

Comment:
⚠️ Can this empty line be removed? We don't usually use them after `if err != nil {`.